src/config/risingwave.toml

```diff
@@ -25,6 +25,7 @@ capacity_mb = 1024
 total_buffer_capacity_mb = 128
 cache_file_fallocate_unit_mb = 512
 cache_meta_fallocate_unit_mb = 16
+cache_file_max_write_size_mb = 4
```

Comment:
Have you tried other parameters in the test, like 1m？

Rest LGTM

Reply:
I did fio benchmark and found that there is no obvious improvement with `4m` than `1m` based on the distribution of read latency. But because there is still a gap between file cache benchmark result and fio result, I can't tell if `1m` or `4m` is still the best chosen after other optimizations (like sharding the meta file lock).

Besides, the default block size of risingwave is set to `1m` for now. So `1m` as max write size basically equals no batching. I think `4m` is a reasonable default configuration for now.

Reply:
I see, thanks for your answer